js/utils/CommonUtils.js

```diff
@@ -105,7 +107,7 @@ define([
 	function renderLink(s, p, d) {
 		var valid = d.INVALID_REASON_CAPTION == 'Invalid' ? 'invalid' : '';
 		var linkClass = getConceptLinkClass(d);
-		return '<a class="' + valid + ' ' + linkClass + '" href=\"#/concept/' + d.CONCEPT_ID + '\">' + d.CONCEPT_NAME + '</a>';
+		return '<a class="' + valid + ' ' + linkClass + '" href=\"#/concept/' + d.CONCEPT_ID + '\">' + filterXSS(d.CONCEPT_NAME) + '</a>';
```

Comment:
I think this should be using the form `if (p === 'display') { return {link text}; } else { return d.CONCEPT_NAME; }`.

This will return the formatted link for display purposes, but other cases it will return just the CONCEPT_NAME value. This will mean the sorting will be performed using just the COCNEPT_NAME, not the link text.